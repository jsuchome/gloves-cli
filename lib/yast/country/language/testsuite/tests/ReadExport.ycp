/**
 * tests for Language.ycp autoyast functions: prepare for cloning
 * Author:	jsuchome@suse.cz
 * $Id: ReadExport.ycp 32769 2006-09-08 09:29:15Z jsuchome $
 */
{
    // testedfiles: Language.ycp

    import "Testsuite";

    map READ = $[
	"sysconfig" : $[
	    "language" : $[
		"RC_LANG"		: "de_DE.UTF-8",
		"INSTALLED_LANGUAGES"	: "en_US,de_DE",
	    ],
	],
	"target" : $[
	    "size"		: 1,
	    "dir" : [
		"language_de_DE.ycp"
	    ],
	    "yast2" : $[
		"de_DE" : [
		    "Deutsch",
		    "Deutsch",
		    ".UTF-8",
		    "@euro",
		    "German"
		],
	    ],
	],
    ];

    map E	= $[
	"target" : $[
	    "bash_output"	: $[],
	],
    ];

    import "Mode";
    Mode::SetMode ("autoinst_config");

    Testsuite::Init ([READ,$[],E], nil);

    import "Language";

    Testsuite::Test (``(Language::Read (true)), [READ,$[],E], nil);

    Testsuite::Test (``(Language::Export ()), [], nil);

    Testsuite::Test (``(Language::Summary ()), [], nil);

    Testsuite::Test (``(Language::Modified ()), [], nil);

}
