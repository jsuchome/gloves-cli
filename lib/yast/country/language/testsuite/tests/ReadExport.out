Read	.target.dir "/usr/share/YaST2/data/languages" ["language_de_DE.ycp"]
Read	.target.yast2 "languages/language_de_DE.ycp" $["de_DE":["Deutsch", "Deutsch", ".UTF-8", "@euro", "German"]]
Execute	.target.bash_output "/usr/bin/locale -a" $[]
Return	true
Return	$["language":"de_DE", "languages":"en_US,de_DE"]
Return	<ul><li>Primary Language: German</li></ul>
Return	false
